core/src/main/java/org/apache/accumulo/core/metadata/schema/Ample.java

```diff
@@ -71,7 +69,7 @@
  * of an effort to remove this specialized code. See #936
  * </ul>
  */
-public interface Ample {
+public interface Ample extends ScanServerRefStore {
```

Comment:
I like this new interface, it brings the code related to scan server refs together.  Maybe instead of having Ample extend it we could have a method on Ample to get it.  The advantage of this is that is makes Ample have less methods at the level so its easier to see what Ample offers.

That could look like the following.

```java
interface Ample {
   ScanServerRefStore scanServerRefs();
}
```


Then the method names on the interface could shortened because they would not need to provide as much context in the name as they would at the top level of ample.  So using it could look like the following w/ an example of a shorter method name.

```java
Ample ample = ... ;

for(ScanServerRefTabletFile ref : ample.scanServerRefs().list()){
   // do some stuff w/ the refs
}
```

Reply:
That sounds good, I can make that change.

Reply:
Changed in c49983c682435f84118b6e508957bb1e54527acc